src/console_cmds.cpp

```diff
@@ -1233,8 +1231,8 @@ DEF_CONSOLE_CMD(ConScript)
 	if (!CloseConsoleLogIfActive()) {
 		if (argc < 2) return false;
 
-		_iconsole_output_file = fopen(argv[1], "ab");
-		if (_iconsole_output_file == nullptr) {
+		_iconsole_output_file = FileHandle::Open(argv[1], "ab");
+		if (_iconsole_output_file.has_value()) {
```

Comment:
I think the logic got inverted here for some reason.
```suggestion
		if (!_iconsole_output_file.has_value()) {
```